### DISCUSSION TITLE
我请问呢

### DISCUSSION THREAD

**Discussion by IXfromThorne** [share121/inter-knot#5177] [灌水]
![Screenshot_2024-09-18-20-17-44-912_com mihoyo hyperion](https://github.com/user-attachments/assets/4362b7e6-ff5f-44df-8c97-768addab962b)
（悲伤噬魂犬.jpg）

**Comment by Fluorine-z** [comment]
这是小程序吗

**Reply by IXfromThorne** [reply]
米游社养成指南
